trench/backends/aws.py

```diff
@@ -13,11 +14,12 @@
 from trench.settings import AWS_ACCESS_KEY, AWS_SECRET_KEY, AWS_REGION
 from botocore.exceptions import ClientError, EndpointConnectionError
 
+
 class AWSMessageDispatcher(AbstractMessageDispatcher):
     _SMS_BODY = _("Your verification code is: ")
     _SUCCESS_DETAILS = _("SMS message with MFA code has been sent.")
 
-    def dispatch_message(self) -> DispatchResponse:
+    def dispatch_message(self, url_name: Optional[str] = None) -> DispatchResponse:
```

Comment:
If we don't care about the args here maybe this would be better:
```suggestion
    def dispatch_message(self, *args, **kwargs) -> DispatchResponse:
```